src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageButtonsBar.vue

```diff
@@ -514,49 +514,60 @@ export default {
 		},
 
 		reminderOptions() {
-			const currentDateTime = moment()
+			const currentDate = new Date()
+			const currentDayOfWeek = currentDate.getDay()
+
+			const nextDay = new Date()
+			nextDay.setDate(currentDate.getDate() + 1)
+
+			const nextSaturday = new Date()
+			nextSaturday.setDate(currentDate.getDate() + (currentDayOfWeek === 6 ? 7 : ((6 + 7 - currentDayOfWeek) % 7)))
+
+			const nextMonday = new Date()
+			nextMonday.setDate(currentDate.getDate() + (currentDayOfWeek === 0 ? 1 : (1 + 7 - currentDayOfWeek)))
```

Comment:
```suggestion
	nextMonday.setDate(currentDate.getDate() + ((8 - currentDayOfWeek) % 7 || 7))
```

Didn't try it though

Reply:
Tested, it does the trick